python/instrumentation/openinference-instrumentation-litellm/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+## 0.1.0 (2024-07-29)
+
+### Features
+
+* Added liteLLM instrumentation for the following functions:
+    - completion()
+    - acompletion()
+    - completion_with_retries()
+    - acompletion_with_retries() **
+    - embedding()
+    - aembedding()
+    - image_generation()
+    - aimage_generation()
+
+** Currently acompletion_with_retries() is buggy on liteLLM's part. I have filed a bug report (https://github.com/BerriAI/litellm/issues/4908) with them but for now, I have commented out the instrumentation and test for acompletion_with_retries()
```

Comment:
I think we should use "we" not "I", or simply use the passive voice for this

Reply:
Changed to passive